packages/frontpage/lib/api/vote.ts

```diff
@@ -0,0 +1,78 @@
+import "server-only";
+import * as db from "../data/db/vote";
+import * as atproto from "../data/atproto/vote";
+import { DataLayerError } from "../data/error";
+import { ensureUser } from "../data/user";
+import { PostCollection } from "../data/atproto/post";
+import { DID } from "../data/atproto/did";
+import { CommentCollection } from "../data/atproto/comment";
+import { invariant } from "../utils";
+import { TID } from "@atproto/common-web";
+
+export type ApiCreateVoteInput = {
+  subject: {
+    rkey: string;
+    cid: string;
```

Comment:
Not used, should be removed.

Reply:
it's used as a strong ref on line 41 for referencing the subject